src/gyazo_client.rb

```diff
@@ -0,0 +1,84 @@
+require 'net/http'
+require 'open3'
+require 'openssl'
+
+class GyazoClient
+
+  def initialize(opts = {})
+    default_opts = {
+      id_file:       ENV['HOME'] + "/.gyazotest.id",
```

Comment:
Sorry, this is my mistake. I leave debug codes.
I will fix  "/.gyazotest.id" -> "/.gyazo.id"